cnxdb/migrations/20170918131724_add_featured_books.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+
+def up(cursor):
+    cursor.execute("""\
+CREATE TABLE featured_books (
```

Comment:
Somehow, a table named `featured_books` exists on production (and staging). There's no trace of it in the cnx-db code, that I can find. Regardless,  we need to deal with that here, since this `up` will fail - In this specific case (production has something weird) I'd say let's rename the existing table out of the way, then rename it back in the down. We have a schema for doing stuff like that  (mostly for datamigrations), so `ALTER TABLE featured_books SET SCHEMA datamigrations` should do it, w/ the matching `SET SCHEMA public` after the `DROP TABLE` in the down()